ui/app/AppLayouts/Chat/panels/communities/CommunityPermissionsSettingsPanel.qml

```diff
@@ -0,0 +1,155 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared.controls.chat 1.0
+
+import "../../layouts"
+
+SettingsPageLayout {
+    id: root
+
+    signal addPermission()
+
+    title: qsTr("Permissions")
+
+    content: Item {
+        QtObject {
+            id: d
+            property int contentWidth: 560 // by design
+        }
+
+        ColumnLayout {
+            anchors.top: parent.top
+            anchors.topMargin: Style.current.padding
+            width: d.contentWidth
+            spacing: 24
+
+            Rectangle {
+                Layout.fillWidth: true
+                color: "transparent"
+                Layout.preferredHeight: contentColumn.implicitHeight + contentColumn.anchors.topMargin + contentColumn.anchors.bottomMargin
+                radius: 16
+                border.color: Theme.palette.baseColor5
+                clip: true
+
+                ColumnLayout {
+                    id: contentColumn
+                    anchors.top: parent.top
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.leftMargin: 16
+                    anchors.rightMargin: anchors.leftMargin
+                    anchors.topMargin: anchors.leftMargin
+                    anchors.bottomMargin: 32
```

Comment:
It could be simplified to:
```suggestion
                    anchors.fill: parent
                    anchors.margins: 16
                    anchors.bottomMargin: 32
```